src/scenes/home/codeSchools/codeSchools.js

```diff
@@ -1,19 +1,24 @@
 import React, { Component } from 'react';
 import LinkButton from 'shared/components/linkButton/linkButton';
 import Section from 'shared/components/section/section';
+import Modal from 'shared/components/modal/modal';
 import ApprovedSchools from './approvedSchools/approvedSchools';
 import PartnerSchools from './partnerSchools/partnerSchools';
 import OnlineSchools from './onlineSchools/onlineSchools';
 import StateSortedSchools from './stateSortedSchools/stateSortedSchools';
+import ScholarshipSchools from './scholarshipSchools/scholarshipSchools';
 import styles from './codeSchools.css';
 
 class CodeSchools extends Component {
   constructor(props) {
```

Comment:
while you're in this file, let's reign in old stuff. All we need from line 13-24 is:

```
state = {
  schools: [],
  modal: false,
  scholarshipSchoolModal: {}
};
```